lib/glific/providers/gupshup/message.ex

```diff
@@ -129,15 +130,7 @@ defmodule Glific.Providers.Gupshup.Message do
     # or maybe a simulator or some test code
     if payload["sender"]["phone"] in [nil, ""] do
       error = "Phone number is blank, #{inspect(payload)}"
-      Logger.error(error)
-
-      stacktrace =
-        self()
-        |> Process.info(:current_stacktrace)
-        |> elem(1)
-
-      Appsignal.send_error(:error, error, stacktrace)
-      raise(RuntimeError, message: error)
+      raise_error_to_appsignal(error)
```

Comment:
We already have a function called Glific.log_error please use that. 
So it could be 
```
Glific.log_error(error)
raise(RuntimeError, message: error)
```
It will help to centralise the errors in future.

Reply:
@pankaj-ag This function was not used in this file before as well. I thought to create a function there as I was using same code. Will make necessary changes